required/latex-lab/latex-lab-block.dtx

```diff
@@ -1173,8 +1171,32 @@
 % \end{environment}
 %
 %
+% \subsubsection{verse environment}
 %
-%
+%  \begin{environment}{verse}
+%  The verse environment has not special tagging currently. It is
+%  defined as a simple standard list and takes the tagging from there.
+%  But it must be redefined so that \cs{itemindent} is correctly set.
```

Comment:
maybe we should explain why this has to be defined inside that hook (because it is class defined)